### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by Advik-B** [Owen-Cochell/cursepy#1]
Python is showing [ModuleNotFoundError](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) when I import cursepy
```py
import cursepy
```
The Full Traceback
```
Traceback (most recent call last):
  File "index.py", line 1, in <module>
    import cursepy
  File "C:\Users\~\AppData\Roaming\Python\Python310\site-packages\cursepy\__init__.py", line 7, in <module>    from cursepy.wrapper import CurseClient
  File "C:\Users\~\AppData\Roaming\Python\Python310\site-packages\cursepy\wrapper.py", line 11, in <module>    from cursepy.handlers.base import HandlerCollection
ModuleNotFoundError: No module named 'cursepy.handlers'
```

**Comment by OwenCochell** [comment]
Apologies for the response delay, my schedule has been very busy as of late.

This error was due to a packaging problem where the sub-modules 'handlers' and 'classes' were not included in the final distribution. I have fixed this and now importing cursepy and it's components work as intended.

If you installed using pip, a simple upgrade should fix it. If not, specifying version 1.1.1 should get you an install that works.
Otherwise, just clone this repository in it's current state. 

Thanks for pointing out this error! I'll close the issue once it's confirmed to work on your end.
